CampaignSvcs/CampaignManager/BidCostProvider.cpp

```diff
@@ -122,12 +121,15 @@ namespace CampaignSvcs
   BidCostProvider::BidCostProvider(
     const String::SubString& directory,
     const Generics::Time& config_timestamp,
-    Generics::TaskRunner* task_runner)
+    Generics::TaskRunner* task_runner,
+    const std::string &fileName)
```

Comment:
variable naming fix required - we use _ as separator for words in variable name

Reply:
const std::string &fileName ->  const std::string& fileName